otherlibs/dune-rpc/private/dune_rpc_private.mli

```diff
@@ -153,6 +153,75 @@ module Where : sig
   end) : S with type 'a fiber := 'a Fiber.t
 end
 
+module Registry : sig
```

Comment:
I don't know if `Registry` is such a good name. Feel free to suggest alternatives.